### PR TITLE
January 25, 2024 - Content Release

### PR DESCRIPTION
**Pull request by salvatoremosca**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-csoar/2024-01-25-content.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    * Added New Action: Get Member Groups
+* [Updated] AWS IAM
+    * Added New Action: Update Access Key
+* [Updated] Slack
+    * Updated action: Ask Question
+* [Updated] AWS EC2
+    * Updated action: Stop Instance
+* [Updated] Atlassian Jira<sup>*</sup>
+    * Several changes have been made. This update introduces BREAKING CHANGES: both the Output Mapping and Input
+      fields have been revised and updated. This version is specific to Jira Server and Data Center.
```
Comment (kimsauce) [inline blog-csoar/2024-01-25-content.md:33]:
```suggestion
    * Several changes have been made. This update introduces BREAKING CHANGES: both the Output Mapping and Input fields have been revised and updated. This version is specific to Jira Server and Data Center. 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@kimsauce, could you approve this PR so we can publish it?

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**